include/SFML/System/Vector2.hpp

```diff
@@ -68,14 +73,138 @@ class Vector2
     ////////////////////////////////////////////////////////////
     template <typename U>
     constexpr explicit Vector2(const Vector2<U>& vector);
+    
+    ////////////////////////////////////////////////////////////
+    /// \brief Length of the vector <i><b>(floating-point)</b></i>.
+    ///
+    /// If you are not interested in the actual length, but only in comparisons, consider using lengthSq().
+    ///
+    ////////////////////////////////////////////////////////////
+    T length() const;
+
+    ////////////////////////////////////////////////////////////
+    /// \brief Square of vector's length <i><b>(floating-point)</b></i>.
+    /// 
+    /// Suitable for comparisons, more efficient than length().
+    ///
+    ////////////////////////////////////////////////////////////
+    constexpr T lengthSq() const;
+
+    ////////////////////////////////////////////////////////////
+    /// \brief Vector with same direction but length 1 <i><b>(floating-point)</b></i>.
+    /// 
+    /// \pre \c *this is no zero vector.
+    ///
+    ////////////////////////////////////////////////////////////
+    [[nodiscard]] Vector2 normalized() const;
+
+    ////////////////////////////////////////////////////////////
+    /// \brief Signed angle from \c *this to \c rhs <i><b>(floating-point)</b></i>.
+    /// 
+    /// \return Angle in degrees in the interval [-180,180].
+    /// The angle determines how much you have to rotate \c *this
+    ///  in either direction, until it is parallel to \c rhs.
+    /// \pre Neither \c *this nor \c rhs is a zero vector.
+    ///
+    ////////////////////////////////////////////////////////////
+    Angle signedAngleTo(const Vector2& rhs) const;
```

Comment:
Calling this `signedAngleTo` insinuates that there might be a different overload returning the unsigned angle when that is not the case. We can refer to angles as simply angles and don't need extra qualifiers on them. The `Angle` abstraction handles messy detail like signed/unsigned and degrees/radians. For good reason we don't say "time in microseconds" because a time is merely a time. We don't need to also speak to a specific detail of the `sf::Time` abstraction. Let the abstractions be abstract.